powersimdata/data_access/data_access.py

```diff
@@ -14,10 +14,22 @@
 from powersimdata.utility import server_setup
 from powersimdata.utility.helpers import CommandBuilder
 
+_dirs = {
+    "tmp": (server_setup.EXECUTE_DIR,),
```

Comment:
using a tuple here makes it so the logic in `get_base_dir` doesn't require an if/else

Reply:
You lost me here. Why a `tuple` is not required for "input" and "output"?

Reply:
They are changed to tuples already in `server_setup.py`